static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js

```diff
@@ -45,7 +45,7 @@ const fillAdvertSlots = (
     start: () => void,
     stop: () => void
 ): Promise<void> => {
-    if (commercialFeatures.dfpAdvertising) {
+    if (commercialFeatures.dfpAdvertising || commercialFeatures.adFree) {
```

Comment:
Doesn't this mean; if `adFree` is `true` then run then add the `googletag` commands?

Also; do we need this check if `dfpAdvertising` already does it?

It seems the checks are at odds with each other; `dfpAdvertising && !adFree` vs `dfpAdvertising || adFree`